python/cuml/tests/test_linear_model.py

```diff
@@ -182,9 +181,8 @@ def test_weighted_linear_regression(
 def test_linear_regression_single_column():
     """Test that linear regression can be run on single column with more than
     46340 rows"""
-    model = cuLinearRegression()
-    with pytest.warns(UserWarning):
-        model.fit(cp.random.rand(46341), cp.random.rand(46341))
+    model = cuml.LinearRegression()
+    model.fit(cp.random.rand(46341)[:, None], cp.random.rand(46341))
```

Comment:
It looks like we now need X to be an array? Is this intentional?

Reply:
The sklearn interface requires a 2D X, and a 1 or 2D y, any case where this was working before was a bug.